internal/go-prompt/history.go

```diff
@@ -2,29 +2,29 @@ package prompt
 
 // History stores the texts that are entered.
 type History struct {
-	histories []string
+	Histories []string
```

Comment:
How about something like `Commands`? `history.Histories` is a bit strange.

Reply:
Renamed as suggested https://github.com/hazelcast/hazelcast-commandline-client/pull/66/commits/fe6bf1e1b2ca7dd456aff72357e77644aca95af7